### PR TITLE
Beta2 updates

### PR DESCRIPTION
**Pull request by vansimke**
This PR includes various theme updates and bug fixes to the site based on [itorry-beta2-updates](https://github.com/dojo/dojo.io/tree/itorrey-beta2-updates) branch.

### CODE REVIEW

**Comment by dylans** [comment]
@vansimke @itorrey it looks like somewhere along the way, all of Torrey's commits were treated as coming from the wrong email address.

**Comment by vansimke** [comment]
Ack! We really need to fix this automation issue... I bet @itorrey's global git profile has been screwed up.

**Comment by vansimke** [comment]
@dylans I think I have the committer name issue resolved now.

File: site/source/api/index.ejs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+description: This is a template used to generate the API index
+title: API Documentation
+---
+<section class="section">
+	<div class="container">
+		<h1 id="API-Documentation"><a href="#API-Documentation" class="headerlink" title="API Documentation"></a>API Documentation</h1>
+		<p><a href="/api/cli/v2.0.0-beta2.3">cli</a><br><a href="/api/compose/v2.0.0-beta1.1">compose</a><br><a href="/api/core/v2.0.0-beta2.4">core</a><br><a href="/api/has/v2.0.0-beta2.3">has</a><br><a href="/api/i18n/v2.0.0-beta2.3">i18n</a><br><a href="/api/interfaces/v2.0.0-beta2.4">interfaces</a><br><a href="/api/loader/v2.0.0-beta2.3">loader</a><br><a href="/api/routing/v2.0.0-beta2.4">routing</a><br><a href="/api/shim/v2.0.0-beta2.4">shim</a><br><a href="/api/stores/v2.0.0-alpha.12">stores</a><br><a href="/api/streams/v2.0.0-alpha.2">streams</a><br><a href="/api/widget-core/v2.0.0-beta2.11">widget-core</a><br><a href="/api/widgets/v2.0.0-beta2.1">widgets</a></p>
+	</div>
+</section>
```
Comment (dylans) [inline site/source/api/index.ejs]: Missing empty line at EOF

File: site/source/docs/index.ejs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: Fundamentals
+layout: docs
+---
```
Comment (dylans) [inline site/source/docs/index.ejs]: New line

File: site/source/tutorials/index.ejs [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
 title: Tutorials
-template: tutorials
+layout: tutorials
 ---
-<h1>Tutorials</h1>
\ No newline at end of file
+<div class="container">
+	<h1>Tutorials</h1>
+</div>
```
Comment (dylans) [inline site/source/tutorials/index.ejs]: New line

File: site/themes/dojo/layout/_partial/archive-post.ejs [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-      <%- partial('post/title', {class_name: 'archive-article-title'}) %>
-    </header>
-  </div>
+	<div class="archive-article-inner">
+		<header class="archive-article-header">
+			<%- partial('post/date', {class_name: 'archive-article-date', date_format: 'MMM D'}) %>
+			<%- partial('post/title', {class_name: 'archive-article-title'}) %>
+		</header>
+	</div>
 </article>
```
Comment (dylans) [inline site/themes/dojo/layout/_partial/archive-post.ejs]: New line

File: site/themes/dojo/layout/_partial/article.ejs [hunk 5]
```diff
@@ -1,43 +1,47 @@
...
+			<div id="disqus_thread">
+				<noscript>Please enable JavaScript to view the <a href="//disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
+			</div>
+		</section>
+		<% } %>
 	</div>
-</section>
-<% } %>
\ No newline at end of file
+</div>
```
Comment (dylans) [inline site/themes/dojo/layout/_partial/article.ejs]: New line

File: site/themes/dojo/layout/tutorials.ejs [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+			var tuts = _.filter(site.pages.data, ['layout', 'tutorial']);
+			tuts = _.orderBy(tuts, ['path']);
+
+			_.each(tuts, function(tut) {
+				// remove index.html from the path to make relative URLs work consistently
+				var urlRaw = tut.path.substr(0, tut.path.length - 10 /* length of 'index.html' */);
+		%>
+			<div class="column is-12-mobile is-6-tablet is-4-desktop is-3-widescreen">
+				<div class="card is-full-height">
+					<!--
```
Comment (dylans) [inline site/themes/dojo/layout/tutorials.ejs]: Should we just remove this comment?

**Comment by agubler** [comment]
@vansimke Is it possible for you to publish to now so we can review the running live site?

**Comment by agubler** [comment]
PS also you have a single conflict now on `site/source/tutorials/005_form_widgets/index.md`

File: site/source/api/index.ejs (hunk 1 above)
Comment (vansimke) [reply site/source/api/index.ejs]: I think this might be coming from the base branch that I forked from @itorrey, but I will look for this in the future.

**Comment by kitsonk** [comment]
Since beta3 is released and there is a PR #198 which effectively supplants this, closing.

**Comment by agubler** [comment]
@kitsonk I don't think that #198 replaces this, I believe this contains the changes required for reference guides, @vansimke could you confirm?

**Comment by vansimke** [comment]
Closing this PR. Will raise another one for the site changes and a third for content review feedback.

**Pull request closed without merging**